cmd/zfs/zfs_main.c

```diff
@@ -307,7 +307,7 @@ get_usage(zfs_help_t idx)
 		    "[filesystem|volume|snapshot] ...\n"));
 	case HELP_MOUNT:
 		return (gettext("\tmount\n"
-		    "\tmount [-flvO] [-o opts] <-a | filesystem>\n"));
+		    "\tmount [-flvO] [-o opts] -r <-a | filesystem>\n"));
```

Comment:
Maybe include `-r` with other short ops?
```suggestion
		    "\tmount [-flrvO] [-o opts] <-a | filesystem>\n"));
```
Or, if it's special enough, bracket it to make clear that it's optional?
```suggestion
		    "\tmount [-flvO] [-o opts] [-r] <-a | filesystem>\n"));
```

Reply:
Yep, this line really needs fixing:

`-r` requires `filesystem`  so it should be `[-r filesystem]`
`-a` has an optional `filesystem`, does this look correct?  `<-a [filesystem] | filesystem>`